5-Accessibility/accessible-form.html

```diff
@@ -0,0 +1,57 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Accesible form</title>
+</head>
+
+<body>
+    <form>
+        <fieldset>
+            <legend>Please enter your contact details</legend>
+            <div>
+                <label for="name">
+            Your name (required):</label>
+                <input type="text" name="name" id="name" required>
+            </div>
+            <div>
+                <label for="email">
+            Email address (required):</label>
+                <input type="email" name="email" id="email" required>
+            </div>
+            <div>
+                <label for="number">
+            Phone number (+57): (optional):</label>
+                <input type="number" name="number" id="number">
+            </div>
+        </fieldset>
+        <fieldset>
+            <legend>Please select the courses that you are interested in (required):</legend>
+            <div>
+                <input type="checkbox" name="web" id="web">
+                <label for="web">Web Developer</label>
```

Comment:
Please include the `aria-attributes`. The same with the other fields.
` <abbr title="required" aria-label="required">*</abbr>`

Reply:
*Update: It's better to use the aria-attribute inside the <input>. You can use the https://validator.w3.org/ to verify it.